containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -589,6 +591,23 @@ public static AthenzJettyContainer createJettyContainer() {
         return container;
     }
 
+    public static void initConfigManager() {
+        System.getProperties().remove("socksProxyHost");
+
+        // First load initial properties (to get parameter store path)
+        String propFile = System.getProperty(AthenzConsts.ATHENZ_PROP_FILE_NAME,
+                getRootDir() + "/conf/athenz/athenz.properties");
+        ConfigProperties.loadProperties(propFile);
+
+        // Manage AWS parameter store configurations (as the first config source, AWS parameters will take precedence over parameters
+        // configured in the properties file.
+        String awsParameterStorePath = System.getProperty(AthenzConsts.ATHENZ_PROP_AWS_PARAM_STORE_PATH, "athenz");
```

Comment:
why do we have a default path here? by default the feature should not be enabled since we may not be running zms in aws.